tgstation.dme

```diff
@@ -8905,6 +8905,7 @@
 #include "modular_zubbers\code\datums\quirks\negative_quirks\numb_override.dm"
 #include "modular_zubbers\code\datums\quirks\negative_quirks\permanent_limp.dm"
 #include "modular_zubbers\code\datums\quirks\negative_quirks\sensitive_snout.dm"
+#include "modular_zubbers\code\modules\quirks\code\negative_quirks\sensative_soles.dm"
```

Comment:
i can't make a suggestion but you need to change `modular_zubbers\code\modules\quirks\code\negative_quirks\sensative_soles.dm` to the correct spelling too